api/query/query.go

```diff
@@ -145,7 +167,7 @@ func (qh queryHandler) loadSummary(testRun shared.TestRun) ([]byte, error) {
 }
 
 func getRedisKey(testRun shared.TestRun) string {
```

Comment:
nit: method name could be more descriptive, e.g. getSummaryFileRedisKey

Reply:
name changed 👍